Installer/package.txt

```diff
@@ -488,6 +488,8 @@ component Extensions
 		rfolder macosx:packaged_extensions/com.livecode.library.objectrepository
 	into [[ToolsFolder]]/Extensions place
 		rfolder macosx:packaged_extensions/com.livecode.library.native.mac.statusmenu
+	into [[ToolsFolder]]/Extensions place
+		rfolder macosx:packaged_extensions/com.livecode.library.getsemver
```

Comment:
@livecodemichael I think there's an extra `get` in the name here

Reply:
@montegoulding Oops! No idea where that came from.